feconf.py

```diff
@@ -693,10 +693,12 @@ def get_empty_ratings():
 DEFAULT_TOPIC_SIMILARITY = 0.5
 SAME_TOPIC_SIMILARITY = 1.0
 
+# While adding another language, please ensure that the
+# languages are in alphabetical order
```

Comment:
I suggest doing this using a list of objects with language information, for example
```
SUPPORTED_SITE_LANGUAGES = [{
    code: 'id',
    name: 'Bahasa Indonesia'
}, {
...
```